Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1564,19 +1573,28 @@ public string GetLocalApiUrl(string host)
                     host,
                     HttpsPort.ToString(CultureInfo.InvariantCulture));
             }
+
             return string.Format("http://{0}:{1}",
                     host,
                     HttpPort.ToString(CultureInfo.InvariantCulture));
         }
 
-        public string GetWanApiUrl(IpAddressInfo ipAddress)
+        public string GetWanApiUrl(IPAddress ipAddress)
         {
-            if (ipAddress.AddressFamily == IpAddressFamily.InterNetworkV6)
+            if (ipAddress.AddressFamily == AddressFamily.InterNetworkV6)
             {
-                return GetWanApiUrl("[" + ipAddress.Address + "]");
+                // Remove the scope id from IPv6 addresses
+                var str = ipAddress.ToString();
```

Comment:
This code is used at least twice, I would wrap it in something like `RemoveScope` to avoid duplication.